aws_secretsmanager_backup.py

```diff
@@ -45,6 +56,14 @@ def main():
                 secrets_client.create_secret(Name=secret['Name'],
                                              SecretString=secret['SecretString'])
 
+def sort_secrets(secrets, sort_mode):
+    sort_key = lambda secret: secret['Name']
+    if sort_mode == 'ascending':
+        return sorted(secrets, key=sort_key)
+    elif sort_mode == 'descending':
+        return sorted(secrets, reverse=True, key=sort_key)
+    else:
```

Comment:
Super-small nitpick: The `else` is not necessary here. Strictly speaking, the `elif` can also be an `if` too, but that's even more pedantic.

Reply:
Hey Dan, 
   Yep you're right, I removed that unnecessary `else` out of there. As for the `if`/`elif`, you're totally right. I tend to like the grouping of using `elif` because it makes it obvious that we're testing a series of related conditions rather than them being disconnected, but this project is yours so if you would like me to change it I am totally happy to. Let me know, thanks!
Phill

Reply:
Looks fine as it is, I think. I doubt this would ever reach the complexity where it's necessary, but if it becomes a bigger project then I'll run pylint/Black over the file and setup CI to do the same.